docs/user-guide/sdks-using.md

```diff
@@ -63,32 +63,25 @@ Download and install the packages.
 
 ### Install SDK from online registry
 
-Define the packages as dependencies in your project and pull them from an online registry.
+Define the packages as dependencies in your project and pull them from the online npm registry.
 
 **Follow these steps:**
 
-1. **(Node.js SDK only)** Install required dependencies for the Node SDK. In a command-line window, navigate to your project directory and issue the following commands:
-npm install @zowe/core-for-zowe-sdk and npm install @zowe/imperative
-    ```
-    npm install @zowe/core-for-zowe-sdk
-    ```
-
-    ```
-    npm install @zowe/imperative
-    ```
-
-2. In command-line window, navigate to your project directory. Issue the following command to install a package from the registry:
+1. In command-line window, navigate to your project directory. Issue the following command to install a package from the registry:
 
    - To import a Node.js package: `npm install <PackageName>`
    - To import a Python package: `pip install <PackageName>`
 
-   *where* `<packageName>` is the name of the package that you want to install, such as `zos-files-for-zowe-sdk`.
+   *where* `<packageName>` is the name of the SDK package that you want to install, such as `zos-files-for-zowe-sdk`.
 
-    For Node.js projects, the package is installed in your `package.json` file.
+    The packages are installed and you can call the APIs from within your project. Node packages are defined in `package.json` in your project. Python packages are installed to a location on your computer such as `/Users/AppData`.
```

Comment:
The location will generally be `$PYTHONPATH/Lib/site-packages`.

Sorry for not being more specific about this before.